integration/spark2/src/main/spark2.2/CarbonSessionState.scala

```diff
@@ -125,6 +132,18 @@ class CarbonSessionCatalog(
   }
 }
 
+
+class CarbonAnalyzer(catalog: SessionCatalog,
```

Comment:
Better move to some common place, don't duplicate

Reply:
Moving to common place is difficult as CarbonAnalyzer takes different parameter in Spark 2.1 amd Spark2.2. In 2.1 it takes CatalystConf and in 2.2 it takes SQLConf.